TypeScript.YAML-tmLanguage

```diff
@@ -37,15 +37,15 @@ repository:
     end: (?=$|[;,])
     patterns:
     - include: '#ternary-operator'
-    - include: '#ternary-expression-type' 
+    - include: '#expression-type' 
 
   ternary-operator:
     begin: (\?)
     end: (:)
     patterns:
-    - include: '#ternary-expression-type' 
+    - include: '#expression-type' 
 
-  ternary-expression-type:
+  expression-type:
```

Comment:
The real problem is that switch statements are not expressions (nor are control statements). But I guess this is fine just to break the circularity.